src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Probably it should be in style.css file, cause it's strange that you applied styles for every element in card styles file